app/src/main/java/com/hadiyarajesh/notex/notification/Counter.kt

```diff
@@ -0,0 +1,5 @@
+package com.hadiyarajesh.notex.notification
+
+object Counter {
```

Comment:
What is this?

Reply:
That I just make for how we handle any action button on notification. For this its just a temporary counter file.